packages/launchpad/cypress/e2e/config-files-error-handling.cy.ts

```diff
@@ -150,3 +150,34 @@ describe('Launchpad: Error System Tests', () => {
     cy.get('[data-testid="error-code-frame"]').should('contain', 'cypress.config.js:4:23')
   })
 })
+
+describe('setupNodeEvents', () => {
+  it('throws an error when in setupNodeEvents updating a config value that was removed in 10.X', () => {
+    cy.scaffoldProject('config-update-non-migrated-value')
+    cy.openProject('config-update-non-migrated-value')
+    cy.visitLaunchpad()
+    cy.findByText('E2E Testing').click()
+    cy.get('h1').should('contain', 'Error Loading Config')
+    cy.percySnapshot()
+  })
+
+  it('throws an error when in setupNodeEvents updating a config value on a clone of config that was removed in 10.X', () => {
+    cy.scaffoldProject('config-update-non-migrated-value-clone')
+    cy.openProject('config-update-non-migrated-value-clone')
+    cy.visitLaunchpad()
+    cy.findByText('E2E Testing').click()
+    cy.get('h1').should('contain', 'Error Loading Config')
+    cy.percySnapshot()
+
+    cy.get('[data-cy="alert-body"]').should('contain', 'integrationFolder')
+  })
+
+  it('throws an error when in setupNodeEvents updating an e2e config value that was removed in 10.X', () => {
+    cy.scaffoldProject('config-update-non-migrated-value-e2e')
+    cy.openProject('config-update-non-migrated-value-e2e')
+    cy.visitLaunchpad()
+    cy.findByText('E2E Testing').click()
+    cy.get('h1').should('contain', 'Error Loading Config')
+    cy.percySnapshot()
```

Comment:
Personally like to assert the actual error using a `cy.get(...).contains(...)`. Not a blocker, but I think those are more clear and easier to debug than Percy snapshots.